### ISSUE TITLE
console - non routable URLs

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1930]
URLs generated by the console webapp are not routable, eg `/ldapadmin/console/home` ends up as a 404, because the webapp is/was located at `/ldapadmin/console`.

**Comment by fvanderbiest** [comment]
RewriteRule are missing for the following paths:
```
/ldapadmin/console/home -> /ldapadmin/console/#/home
/ldapadmin/console/users/* ->  /ldapadmin/console/#users/*
/ldapadmin/console/orgs/* -> /ldapadmin/console/#/orgs/*
/ldapadmin/console/groups/* -> /ldapadmin/console/#/groups/*
/ldapadmin/console/org/* -> /ldapadmin/console/#org/*
/ldapadmin/console/logs -> /ldapadmin/console/#/logs
/ldapadmin/console/analytics/*-> /ldapadmin/console/#/analytics/*
```

**Comment by fvanderbiest** [comment]
Fixed with:
```
RewriteCond %{REQUEST_URI} !^/ldapadmin/console/public/.*$
RewriteCond %{REQUEST_URI} ^/ldapadmin/console/([home|users|org|orgs|groups|logs|analytics].*)$
RewriteRule .* /ldapadmin/console/#/%1 [NE,R,L]
```

**Comment by fvanderbiest** [comment]
Fixed in the doc with 42b26032a604818873da971b126386e11c0ed1cb

Release notes now mention it with a6a234c04d3a7eb2340bc8038dbc2db9dfcf198a

**Comment by fvanderbiest** [comment]
For the reference, an equivalent haproxy cfg (georchestra 17.12):
```
frontend 443
    acl console_routes path_beg /ldapadmin/console/home /ldapadmin/console/users /ldapadmin/console/org /ldapadmin/console/orgs /ldapadmin/console/roles /ldapadmin/console/logs /ldapadmin/console/analytics
    http-request redirect code 301 location "https://%[hdr(host)]%[url,regsub(^/ldapadmin/console/,/ldapadmin/console/#/,g)]" if console_routes
```
